cmd/ddev/cmd/import-files.go

```diff
@@ -29,6 +29,8 @@ func NewImportFileCmd() *cobra.Command {
 			The destination directories can be configured in your project's config.yaml
 			under the upload_dirs key. If no custom upload directory is defined, the app
 			type's default upload directory will be used.
+
+			If importing a Tar or Zip archive, the archive should contain only the *contents* of the top-level target directory. For example, an archive of the Drupal 'sites/default/files' directory should contain only the contents of 'sites/default/files', not the 'files' directory.
```

Comment:
I guess IMO that's too much information. It's very specific about how to create the tarball.

Reply:
aside the detail about content and contents i've asked over in discord.  

> If using a Tar or Zip archive, the archive should contain only the files inside the files directory. For example in a Drupal site with files at sites/default/files, the archive should **not* contain the files directory, just the contents of the files directory

the second sentence took me one or two times until i understood. i wonder if it would make sense to switch the order  to what the goal is and then what situation should be avoided. i suppose the wording could still be improved

```For example in a Drupal site with files at sites/default/files, the archive should only contain the contents of that `files directory`. You shouldn't end up with a single `files` directory on unarchive containing all the contents.```

And i wonder if it would make sense to shorten: 

> The easiest way to do this with the tar command is to cd into the directory and create the archive there

I would simply strike "the easiest way to do this.." and use something like 

```
To export the files directory use: 
```

or not sure what matt recommends as the common phrase for pointing at one or two lines of terminal input.